lua/lint/util.lua

```diff
@@ -22,4 +22,14 @@ function M.offset_to_position(lines, offset)
 end
 
 
+function M.split(str, sep)
```

Comment:
There is already `vim.split` or `vim.fn.split`, I think the former behaves pretty much the same. Could that be used instead?

Reply:
Ah, completely missed that. I tried looking up a `split` command but it did not show up. I have made the necessary changes to use the `vim.fn.split` function.